modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/TableKeyDetails/index.jsx

```diff
@@ -219,7 +220,17 @@ const TableKeyDetails = ({activationKeys, setValueToCopyToClipboard}) => {
 				>
 					<div className="col-5">
 						<p className="text-neutral-8 text-paragraph-sm">
-							Instance Size
+							{i18n.translate('instance-size')}
+
+							<PopoverIconButton
+								articleAccountSupport={{
+									text: i18n.translate(
+										'learn-more-about-the-instance-sizing'
+									),
+									url:
+										'https://help.liferay.com/hc/en-us/articles/360025762932#h_01H3YAFQ7B5H1ZHCXZX7E64SJ1',
```

Comment:
We need to pass this as argument for Client Extension (Remote App) and get this information from context

See this PR as example:

https://github.com/pt-liferay-solutions/liferay-portal/pull/4048/files

You can consult @samuelleolopes if you have any questions

```suggestion
										'https://help.liferay.com/hc/en-us/articles/360025762932#h_01H3YAFQ7B5H1ZHCXZX7E64SJ1',
```